### PR TITLE
tar

### PR DESCRIPTION
**Pull request by fiquick**

### CODE REVIEW

**Comment by fiquick** [comment]
[.tar files are compatible for importing to AuraDB and AuraDS](https://trello.com/c/l1g7TwDY/392-tar-files-are-compatible-for-importing-to-auradb-and-aurads)

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-aura-523.surge.sh

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-aura-523.surge.sh

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-aura-523.surge.sh

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-aura-523.surge.sh

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-aura-523.surge.sh

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-aura-523.surge.sh

File: modules/ROOT/partials/import-database.adoc [hunk 1]
```diff
@@ -1,22 +1,21 @@
...
 
-You can use the *import database* process to import either a _.backup_ file or a _.dump_ file.
-This process, however, only works for _.backup_ and _.dump_ files under 4GB.
+Use the *import database* process in the Aura console UI to import a _.backup_, _.dump_ or _.tar_ file as long as it’s less than 4GB. 
 
-If the size of the _.backup_ or _.dump_ file exported from a database is greater than 4GB, you must use the <<_neo4j_admin_database_upload>> method.
+Use the <<_neo4j_admin_database_upload>> command if the size of the  _.backup_, _.dump_ or _.tar_ file exported from a database larger than 4GB.
 
-When importing a _.backup_ or _.dump_ file, it is highly recommended that it is version 4.4 or higher.
+It is recommended that the data in the file is exported from a Neo4j database running version 4.4 or higher.
```
Comment (AlexicaWright) [inline modules/ROOT/partials/import-database.adoc:7]:
```suggestion
It is highly recommended that the _.backup_,  _.dump_ , or _.tar_file, is of version 4.4 or higher.
```

**Review by AlexicaWright — COMMENTED** [review]
One small suggestion.

File: modules/ROOT/partials/import-database.adoc [hunk 2]
```diff
@@ -1,22 +1,21 @@
...
 
-You can use the *import database* process to import either a _.backup_ file or a _.dump_ file.
-This process, however, only works for _.backup_ and _.dump_ files under 4GB.
+Use the *import database* process in the Aura console UI to import a _.backup_, _.dump_ or _.tar_ file as long as it’s less than 4GB. 
 
-If the size of the _.backup_ or _.dump_ file exported from a database is greater than 4GB, you must use the <<_neo4j_admin_database_upload>> method.
+Use the <<_neo4j_admin_database_upload>> command if the size of the  _.backup_, _.dump_ or _.tar_ file exported from a database larger than 4GB.
 
-When importing a _.backup_ or _.dump_ file, it is highly recommended that it is version 4.4 or higher.
+It is highly recommended that the _.backup_,  _.dump_ , or _.tar_file, is of version 4.4 or higher.
```
Comment (AlexicaWright) [inline modules/ROOT/partials/import-database.adoc:7]:
```suggestion
It is highly recommended that the _.backup_,  _.dump_ , or _.tar_ file, is of version 4.4 or higher.
```

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-aura-523.surge.sh

**Review by AlexicaWright — APPROVED** [review]

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-aura-523.surge.sh

**Pull request merged**